Kernel/Syscalls/pipe.cpp

```diff
@@ -43,7 +43,7 @@ ErrorOr<FlatPtr> Process::sys$pipe(Userspace<int*> pipefd, int flags)
             reader_fd_allocation.fd,
             writer_fd_allocation.fd,
         };
-        if (copy_to_user(pipefd, fds_for_userspace, sizeof(fds_for_userspace)).is_error()) {
+        if (copy_n_to_user(pipefd, fds_for_userspace, 2).is_error()) {
             fds[reader_fd_allocation.fd] = {};
```

Comment:
(thereʼs a nice comment for this  in `socket.cpp:432` – care to put it also here?)